### PR TITLE
INDY-2102: Change GET_AUTH_RULE reply format

### PR DESCRIPTION
**Pull request by Toktar**

### CODE REVIEW

**Comment by Toktar** [comment]
test this please

File: indy_node/test/auth_rule/test_get_auth_rule.py [hunk 1]
```diff
@@ -143,12 +135,58 @@ def test_get_all_auth_rule_transactions_after_write(looper,
...
+                                           sdk_pool_handle)
+
+    # prepare action key
+    dict_key = dict(auth_rules[0][1][RESULT][DATA][0])
+    dict_key.pop(CONSTRAINT)
+
+    # prepare "operation" to send AUTH_RULE txn
+    op = dict(auth_rules[0][1][RESULT][DATA][0])
+    op[TXN_TYPE] = AUTH_RULE
+    op[CONSTRAINT] = constraint
```
Comment (ashcherbakov) [inline indy_node/test/auth_rule/test_get_auth_rule.py]: Can we send AUTH_RULE with the same Constraint (that it don't override ` op[CONSTRAINT] `)?

Comment (Toktar) [reply indy_node/test/auth_rule/test_get_auth_rule.py]: Yes. Are we need to add this test?

Comment (ashcherbakov) [reply indy_node/test/auth_rule/test_get_auth_rule.py]: Yes, would be great, to make sure that we can use the GET output as-is for the SET input

**Pull request merged**